src/skeleton/styled-components.js

```diff
@@ -0,0 +1,20 @@
+
+import * as React from 'react';
+import {styled } from '../styles/index.js';
+
+
+
+export const StyledRoot = styled('div', {
+    display: 'flex',
+    flexDirection: 'row',
```

Comment:
why does this need to be a flex layout?

Reply:
I changed the default flexDirection to 'column'.  I thought  'flex' can help better arrange rows in a container. What's your opinion? Do I need it?